client/gutenberg/extensions/presets/jetpack/editor.js

```diff
@@ -4,4 +4,5 @@
  * Internal dependencies
  */
 import 'gutenberg/extensions/markdown/editor';
+import 'gutenberg/extensions/related-posts';
```

Comment:
We're renaming everything to `editor.js` (https://github.com/Automattic/wp-calypso/pull/27119) — good if you rebase and rename here, too.

Reply:
(Done by me)

Reply:
Thanks; although I'm not sure why it was renamed to `view` and not `editor`. Fixed in https://github.com/Automattic/wp-calypso/pull/26530/commits/67a68f67c702e315ce0332a29dde90aded65811d.